devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/tasks/QuarkusDev.java

```diff
@@ -1,5 +1,9 @@
 package io.quarkus.gradle.tasks;
 
+import static io.quarkus.analytics.dto.segment.ContextBuilder.CommonSystemProperties.GRADLE_VERSION;
+import static io.quarkus.analytics.dto.segment.TrackEventType.*;
+import static java.util.Collections.*;
```

Comment:
Same here, let's get rid of the star imports.

Reply:
sure

Reply:
On a follow up PR